### PR TITLE
Documentation improvement about Running Metabase on Debian as a service with nginx

### PR DESCRIPTION
**Pull request by MrCirca**
Hello,
I have added a newer - modern practice to create metabase service with systemd unit file, that is preferred using systemd.

Thank you

### CODE REVIEW

**Comment by omegadeep10** [comment]
Should there be some text about using `systemctl enable <servicename>` to ensure metabase starts on boot?

**Comment by MrCirca** [comment]
Is it ok;

**Review by omegadeep10 — CHANGES_REQUESTED** [review]
metabase is slightly misspelled after `systemctl enable` I think

**Comment by MrCirca** [comment]
Ok i am sorry. Resolved

**Comment by omegadeep10** [comment]
Oh also, I'd recommend `Type=oneshot` instead of `Type=simple`. https://stackoverflow.com/questions/39032100/what-is-the-difference-between-systemd-service-type-oneshot-and-simple

What do you think? Perhaps there are reasons for type simple I'm unaware of, let me know.

**Comment by MrCirca** [comment]
I have to try it another time....Maybe tommorow! The current unit file, works fine for me

**Comment by omegadeep10** [comment]
Yep, makes sense. This looks good to me. I have Metabase running using Debian and Systemd as well. I'll have to check what I set `Type` to :)

File: docs/operations-guide/running-metabase-on-debian.md [hunk 1]
```diff
@@ -180,14 +180,39 @@ Now, it's time to register our Metabase service with `systemd` so it will start
     # ensure unprivileged deploy user owns log (or it won't be able to write)
     $ sudo chown <your_deploy_user>:<your_deploy_user> /var/log/metabase.log
 
-    # add to default services
-    $ sudo update-rc.d metabase defaults
+#### Create a service  systemd unit file
```
Comment (wvengen) [inline docs/operations-guide/running-metabase-on-debian.md]: I think this sentence could be improved, e.g. _Create a systemd service_. There's also a double space now.

**Comment by walterl** [comment]
Closed due to inactivity, overlap with merged #9037.

**Pull request closed without merging**